test/moyasar_test.rb

```diff
@@ -17,7 +17,7 @@ def test_should_accept_api_key
 
   def test_request_should_throw_exception_if_key_is_nil
     stub_server_request(:payments, status: 401)
- 
+
     assert_raises Moyasar::AuthenticationError do
       Moyasar.request(:get, '/v1/payments')
```

Comment:
This is a spot for inspection when enhancing the test setup ..
Sometimes fails locally too.